pkg/kv/scan.go

```diff
@@ -43,8 +43,12 @@ func (b *PrefixIterator) Close() {
 }
 
 // ScanPrefix returns an iterator on store that scan the set of keys that start with prefix
-func ScanPrefix(ctx context.Context, store Store, prefix []byte) (EntriesIterator, error) {
-	iter, err := store.Scan(ctx, prefix)
+func ScanPrefix(ctx context.Context, store Store, prefix, after []byte) (EntriesIterator, error) {
+	start := prefix
+	if len(after) > 0 {
+		start = after
+	}
+	iter, err := store.Scan(ctx, start)
```

Comment:
From the implementation I understand that `after` is a full key and not a relative one under prefix. A documentation of this is missing. Also - should we validate it?

Reply:
We should probably sanitize the value when getting it from the user, I will add it and the documentation